apps/test-suite/tests/FileSystem.js

```diff
@@ -24,7 +24,6 @@ export async function test({ describe, expect, it, ...t }) {
         it('throws out-of-scope exceptions', async () => {
           const p = FS.documentDirectory;
 
-          await throws(() => FS.getInfoAsync(p + '../hello/world'));
```

Comment:
it's useful to add comments on prs for changes like this where the motivation for the change is unclear, makes it easier to review

Reply:
Gotcha - looks like the return type for this method has changed - it no longer throws when empty!